docs/overview/office-add-in-availability.md

```diff
@@ -693,6 +688,40 @@ If a table cell contains an asterisk ( * ), that means we’re working on it. Fo
 
 <br/>
 
+## Project
+
+<table style="width:80%">
+  <tr>
+    <th>Platform</th>
+    <th>Extension points</th>
+    <th>API requirement sets</th>
+    <th><a href="https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets"><b>Common APIs</b></a></th>
+  </tr>
+  <tr>
+    <td>Office for Windows 2013</td>
+    <td> - Taskpane</td>
+    <td> - <a href="https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets">DialogApi 1.1</a></td>
+    <td> - Selection<br>
+         - TextCoercion</td>
+  </tr>
+  <tr>
+    <td>Office for Windows 2016</td>
+    <td> - Taskpane</td>
+    <td> - <a href="https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets">DialogApi 1.1</a></td>
+    <td> - Selection<br>
+         - TextCoercion</td>
+  </tr>
+  <tr>
+    <td>Office for Windows 2019</td>
```

Comment:
For consistency with other mentions on this page, this should be rephrased as:  Office 2019 for Windows